### ISSUE TITLE
Build a minimal base image for gitpod

### ISSUE THREAD

**Issue by dclong** [gitpod-io/gitpod#932]
Can we have a minimal base image for gitpod for more experience Docker users to extend from? Below is a related issue. 
https://github.com/gitpod-io/gitpod/issues/931

**Comment by RDIL** [comment]
Just a note: my issue is #930 (typo)

**Comment by JesterOrNot** [comment]
what did you have in mind?

**Comment by AlexTugarev** [comment]
@dclong you can start with a plain alpine or debian based image if you want. 🙌 

https://gitpod.io/#https://github.com/AlexTugarev/test.repo/tree/custom-image – this is an example which uses `node:alpine`.

You can iterate on your own Dockerfile, and get inspired by https://github.com/gitpod-io/workspace-images. Find the docs at https://www.gitpod.io/docs/42_config_docker/

**Comment by dclong** [comment]
@AlexTugarev @JesterOrNot Thank you very much! It turned out that I misunderstood the gitpod Docker images. I though one must extend from `gitpod/workspace-full`. Being able to use a customize Docker image as the base image is indeed very convenient!

I tried to use `ubuntu:19.04` as the base image and it worked however, it didn't work for me when I used my own customized image `dclong/python`. It is likely due to the fact that I have `ENTRYPOINT ["/scripts/sys/init.sh"]` in the Dockerfile. So may we document the detailed requirements on the based Docker image.
